python_scripts/make_merged_CDE_table.py

```diff
@@ -0,0 +1,50 @@
+#!/bin/env python
```

Comment:
Is it possible to write a unit test for this file? You can create a new ```test``` directory within ```python_scripts``` and put ```test_make_merged_CDE_table.py``` there. [Here](https://github.com/biocore/WGS-HGT/tree/master/benchmark/tests) are some example testing scripts and I'll also submit shortly a PR for testing the Kraken results to BIOM table script.